ote/src/clj/ote/services/transport.clj

```diff
@@ -105,6 +106,9 @@
 (defn- save-transport-operator [db data]
   (upsert! db ::transport-operator/transport-operator data))
 
+(defn- user-has-rights-to-save-service [operator service]
```

Comment:
Could we have more generic access control functions that can be reused in every form? ->  user-has-rights-to-service [operator service].
Getting ids from operator and services seems to be using quite specific paths. Could we have generic helper fns for those tasks? e.g get-id

Reply:
You have to give me example how to do this.
I have no idea how to create more generic method for access rights.

I moved access right check to cljc folder. Now it is possible to take into account in ui side too (not implemented, but possible).